tests/DCPS/ContentFilteredTopic/ContentFilteredTopicTest.cpp

```diff
@@ -348,6 +344,163 @@ bool run_unsignedlonglong_test(const DomainParticipant_var& dp,
   return true;
 }
 
+/*
+ * NOTE: There is a test almost exactly like this in the QueryConditon test to
+ * test the same situation with QueryConditions.
+ */
+bool run_single_dispose_filter_test(const DomainParticipant_var& dp,
+  const MessageTypeSupport_var& ts, const Publisher_var& pub,
+  const Subscriber_var& sub,
+  const char* filter, bool expect_dispose)
+{
+  Duration_t max_wait_time = {3, 0}; // 3 sec
+  ReturnCode_t ret;
+
+  // Create Topic
+  CORBA::String_var type_name = ts->get_type_name();
+  Topic_var topic = dp->create_topic(
+    "Messenger Topic", type_name,
+    TOPIC_QOS_DEFAULT, 0, DEFAULT_STATUS_MASK);
+
+  // Create ContentFilteredTopic
+  ContentFilteredTopic_var cft = dp->create_contentfilteredtopic(
+    expect_dispose ? "Safe Filtered Messenger Topic" : "Unsafe Filtered Messenger Topic",
+    topic, filter, StringSeq());
+  if (!cft) {
+    cerr << "ERROR: run_dispose_filter_test: creating ContentFilteredTopic failed" << endl;
```

Comment:
Should these ERROR statements match the function name, run_single_dispose_filter_test?